tabular/tests/unittests/models/test_rf.py

```diff
@@ -25,3 +25,14 @@ def test_rf_regression(fit_helper):
     )
     dataset_name = 'ames'
     fit_helper.fit_and_validate_dataset(dataset_name=dataset_name, fit_args=fit_args)
+
+
+def test_rf_compile_binary(compile_helper):
```

Comment:
This isn't actually compiling to ONNX since skl2onnx isn't installed. It is simply failing to import onnx and using native. Need to add the onnx dependency to tabular's setup.py `tests` extra dependencies. https://github.com/awslabs/autogluon/blob/master/tabular/setup.py#L67-L69

Reply:
Added `skl2onnx` into extra dependencies.

NOTE: When installing skl2onnx, it would also introduce `onnxruntime-gpu` package. We are now explicitly taking CPU provider in the onnx inference session.